Kinvey.Core/Auth/User.cs

```diff
@@ -543,22 +569,50 @@ static public void LoginWithAuthorizationCodeLoginPage(string redirectURI, Kinve
 		/// <param name="password">Password for authentication</param>
 		/// <param name="redirectURI">The redirect URI to be used for parsing the grant code</param>
 		/// <param name="ct">[optional] CancellationToken used to cancel the request.</param>
+		[Obsolete("This method has been deprecated.  Please use LoginWithMIC() instead.")]
 		static public async Task LoginWithAuthorizationCodeAPIAsync(string username, string password, string redirectURI, AbstractClient userClient = null, CancellationToken ct = default(CancellationToken))
 		{
+			await LoginWithMIC(username, password, redirectURI, null, userClient, ct);
+		}
+
+		/// <summary>
+		/// Performs MIC Login authorization through an API.
+		/// </summary>
+		/// <param name="username">Username for authentication</param>
+		/// <param name="password">Password for authentication</param>
+		/// <param name="redirectURI">The redirect URI to be used for parsing the grant code</param>
+		/// <param name="ct">[optional] CancellationToken used to cancel the request.</param>
+		static public async Task LoginWithMIC(string username, string password, string redirectURI, string micID = null, AbstractClient userClient = null, CancellationToken ct = default(CancellationToken))
```

Comment:
@vinaygahlawat  Is there a reason we continue to maintain sync and async `LoginWithMIC` methods? Can we get rid of one of the two?